confidential-assets/src/confidentialTransfer.ts

```diff
@@ -58,7 +58,11 @@ export class ConfidentialTransfer {
 
   confidentialAmountToTransfer: ConfidentialAmount;
 
-  confidentialAmountAfterTransfer: ConfidentialAmount;
+  encryptedAmountToTransfer: TwistedElGamalCiphertext[];
+
+  confidentialBalanceAfterTransfer: ConfidentialAmount;
+
+  encryptedBalanceAfterTransfer: TwistedElGamalCiphertext[];
```

Comment:
Can we get some comments on these? I think I get it but someone new probably wouldn't know what is different between confidential x and encrypted x.

Reply:
Will soon be refactoring this into a EncryptedAmount class that will combine ConfidentialAmount and TwistedElGamalCiphertext[] fyi.  

Plan to get rid of the use of ConfidentialAmount as attribute throughout SDK.